README.md

```diff
@@ -1,6 +1,8 @@
 # React Structural Metadata Editor
 
-A React component which displays structural metadata about an ingested work, and a visualized waveform to help navigating sections of the audio waveform. A user can add, edit and delete headers and timespans within the structural metadata, and save the record into a consuming application.
+A React component which displays structural metadata about an ingested audio/video file, and displays a visualized waveform to help navigating sections of the audio waveform. A user can add, edit and delete headers and timespans within the structural metadata, and save the record into a consuming application.
+
+**Note**: We are not currently publishing this package to NPM, but rather consuming it directly via a Github repository URL address. See **Deployment** notes below for more info and example code.
 
```

Comment:
@adamjarling Do you think it's better if we link the `Deployment` section here?

Reply:
Yes, good idea.